Jenkinsfile.continuous

```diff
@@ -0,0 +1,50 @@
+// This pipeline is used to build and publish new versions of the stack when changes are merged into
+// the master branch.
+pipeline {
+    agent { label 'upbound-gce' }
+
+    options {
+        disableConcurrentBuilds()
+        timestamps()
+    }
+
+    // Checks for unprocessed changes in the repo once per day
+    // 'H' allows Jenkins to choose the time to balance the load
+    triggers {
+        pollSCM('H H * * *')
+    }
+
+    environment {
+        DOCKER = credentials('dockerhub-upboundci')
+        CROSSPLANE_CLI_RELEASE = 'v0.2.0'
+    }
+
+    stages {
+        stage('Prepare') {
+            steps {
+                sh 'mkdir bin'
+                sh "curl -sL https://raw.githubusercontent.com/crossplaneio/crossplane-cli/${CROSSPLANE_CLI_RELEASE}/bootstrap.sh | env PREFIX=${WORKSPACE} RELEASE=${CROSSPLANE_CLI_RELEASE} bash"
```

Comment:
```suggestion
                sh "curl -sL https://raw.githubusercontent.com/crossplane/crossplane-cli/${CROSSPLANE_CLI_RELEASE}/bootstrap.sh | env PREFIX=${WORKSPACE} RELEASE=${CROSSPLANE_CLI_RELEASE} bash"
```

Reply:
I still need to update the go mod portion of `crossplane-cli` but it seems that it likely won't affect this.

Reply:
Awesome - thanks! Yeah, the releases all use built go binaries for the go-based tools, so it shouldn't affect them once they're downloaded.